modules/cluster-logging-uninstall.adoc

```diff
@@ -60,9 +62,23 @@ Do not delete the `openshift-operators-redhat` project if other global operators
 
 .. Confirm the deletion by typing `openshift-operators-redhat` in the dialog box and click *Delete*.
 
-. Optional: Remove any OpenShift Logging persistent volume claims (PVC):
+[WARNING]
+====
+Releasing or deleting PVCs can delete PVs and cause data loss.
+====
+
+. To keep the PVCs for reuse with other pods, keep the labels or PVC names that you need to reclaim the PVCs.
+
+. Optional: If you do not want to reclaim the PVCs, you can delete them.
++
+If the value of the PV `persistentVolumeReclaimPolicy` property is `delete`, deleting the PVC also deletes the corresponding PV. This action releases the backing storage and removes all data, such as collected log data.
 
 .. Switch to the *Storage* -> *Persistent Volume Claims* page.
 
 .. Click the Options menu {kebab} next to each PVC and select *Delete Persistent Volume Claim*.
 
+.. If the value of the PV `persistentVolumeReclaimPolicy` property is `retain`, use `$ kubectl delete pv --all` or `$ kubectl delete pv <pv-name>` to delete the PVs.
```

Comment:
@rolfedh I suggest to use `oc` rather than `kubectl`.

Reply:
@rolfedh `$ kubectl delete pv --all` is too dangerous, it will delete all the PVs in the cluster since PV is not a namespaced api-resource, could we remove this command?

Reply:
Thanks, @QiaolingTang. I've updated the content accordingly.